mc2/controllers/docker/templates/docker/inclusion_tags/controller_object.html

```diff
@@ -9,8 +9,29 @@
       <h3 class="widget-user-username">{{controller.name}}</h3>
       <div class="widget-user-desc">{{controller.description}}</div>
       <div class="widget-user-desc">
-        {{controller.app_id }}
-        <span class="label bg-{% if controller.state == 'done' %}green{% else %}orange{% endif %}">{{controller.state}}</span>
+          {% with controller.get_status as status %}
```

Comment:
@jerith to your point in slack - I agree that this should possibly be moved to the view. The other alternative is to pull it in using Ajax. I opted to keep the initial exploration for calling `get_status` in the template because that's not the core focus of the work - the core focus is the API and so happy for the initial implementation to be slightly crude on the template.